core/service/transaction.ts

```diff
@@ -13,20 +14,22 @@ import TransactionPool from './transactionPool';
 import TransactionQueue from './transactionQueue';
 import { transactionSortFunc, getTransactionServiceByType, TRANSACTION_BUFFER_SIZE } from 'core/util/transaction';
 import BUFFER from 'core/util/buffer';
+import { SALT_LENGTH } from 'core/util/const';
 
 export interface ITransactionService<T extends IAsset> {
-    getBytes(trs: Transaction<T>): Buffer;
+    getBytes(trs: TransactionModel<T>): Buffer;
```

Comment:
Why change from `Transaction` to `TransactionModel`? `getBytes` is called on verified transaction same as `calculateFee`, it should be `Transaction` entity